### ISSUE TITLE
Custom socket options

### ISSUE THREAD

**Issue by jhen0409** [zalmoxisus/remote-redux-devtools#1]
https://github.com/zalmoxisus/remotedev-app/blob/master/src/app/constants/socketOptions.js

I want to customize `socketOptions`, and I can run socketcluster server myself.
If this is in progress, then I will close it. :)

**Comment by jhen0409** [comment]
Also, we can add socketcluster server to `bin`? or another CLI tool?

It may be used like this:
```js
"scripts": {
  "redux": "remote-redux-devtools --host=localhost --port=5678"
}
```

**Comment by zalmoxisus** [comment]
I do want to implement this, and like the idea of CLI, with host `0.0.0.0`, which can be exposed on LAN (we ought to figure out how to do that and specify it in the README), so we may test the app also on the mobile device not only on the emulator. Before we implement it, we need to add a customization page on [our app](https://github.com/zalmoxisus/remotedev-app) to be able to specify host and port here as well.

**Comment by jhen0409** [comment]
:+1: I very hope this implements, because I don't use `remotedev.io`.

**Comment by zalmoxisus** [comment]
Actually `remotedev.io` is a domain I registered only for conveniency, to ship this module as "turn and go", so you may use it freely without any obligations.

Of course, we'll have the ability to use localhost as it is faster and it will help to not overload my server for development, and the solution with CLI is convenient enough.

The main advantage of the remote domain will come for the production, so we'll be able to turn the devtools on, when it is necessary, and debug remotely.

**Comment by zalmoxisus** [comment]
Implemented in `v0.0.5`. Just run `remotedev --hostname=localhost --port=5678`. In [`store/configureStore.js`](https://github.com/zalmoxisus/remote-redux-devtools#storeconfigurestorejs) use `devTools({ hostname: 'localhost', port: 5678 })`.

Also, for now, you have to change hostname and port in [monitor app](https://github.com/zalmoxisus/remotedev-app/blob/master/src/app/constants/socketOptions.js). Later we'll add a configuration page there (it's just an UI issue).

**Comment by zalmoxisus** [comment]
Implemented in [remotedev-server](https://github.com/zalmoxisus/remotedev-server). The reasons for extracting it as a separate package are:
1. We do not want installing unnecessary dependencies like `socketcluster` and `express` if the localserver is not used.
2. We can use it in other projects like [remotedev](https://github.com/zalmoxisus/remotedev).
3. As it is optional, better to describe usage elsewhere.

Anyway we'll keep it here in `0.0.x` as it is a breaking change.
